### PR TITLE
Add MonoGame.WindowsDX SDK-style csproj

### PR DESCRIPTION
**Pull request by Jjagg**
This pull request intends to replace the Protobuild generated project on Windows. Follow up of #6650.
See #6768 for discussion.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -80,7 +80,6 @@ Tools/MonoGame.Build.Tasks/MonoGame.Build.Tasks.*.csproj
 ThirdParty/Lidgren.Network/*.csproj
 Test/*.csproj
 !Test/MonoGame.Test.XNA.csproj
-/*.sln
```
Comment (cra0zy) [inline .gitignore]: Don't remove this line from gitignore, we still need it, instead just force add .sln file you need.

File: Build/Module.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <DefaultAction>Generate</DefaultAction>
   <DefaultLinuxPlatforms>Android,Angle,Linux,WindowsGL</DefaultLinuxPlatforms>
   <DefaultMacOSPlatforms>Angle,MacOS,iOS,WindowsGL,Android,tvOS,Linux</DefaultMacOSPlatforms>
-  <DefaultWindowsPlatforms>Android,Angle,Linux,Windows,WindowsGL,WindowsUniversal,iOS,tvOS</DefaultWindowsPlatforms>
```
Comment (cra0zy) [inline Build/Module.xml]: You have broken Pipeline Tool generation by removing this.

Comment (Jjagg) [reply Build/Module.xml]: We can keep it until we migrate tools, will undo.

File: MonoGame.Framework.WindowsDX.sln [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
```
Comment (Jjagg) [inline MonoGame.Framework.WindowsDX.sln]: Oops, forgot the project 😅

Comment (cra0zy) [reply MonoGame.Framework.WindowsDX.sln]: The important thing is, it builds XD

**Comment by cra0zy** [comment]
You got squashed! Also merged!

**Pull request merged**